src/flows/components/FlowPage.tsx

```diff
@@ -24,12 +24,13 @@ import 'src/flows/style.scss'
 
 const FlowFromRoute = () => {
   const {id} = useParams<{id: string}>()
-  const {change} = useContext(FlowListContext)
+  const {change, flows, currentID} = useContext(FlowListContext)
 
   useEffect(() => {
     change(id)
   }, [id, change])
 
+  document.title = flows[currentID]?.name ?? 'Notebooks'
```

Comment:
I think we want to mimic how dashboards handles this with https://github.com/influxdata/ui/blob/master/src/shared/utils/pageTitles.ts

Like this: https://github.com/influxdata/ui/blob/master/src/dashboards/components/dashboard_index/DashboardsIndex.tsx#L67

Reply:
Way to make this harder, GENE

Reply:
;)